travvy-server/src/controllers/DetailsController.js

```diff
@@ -0,0 +1,16 @@
+
+// Imports attractions model from Attractions.js
+const Attractions = require('../models').Attractions
+
+// This is the logic for queriying database for attraction information for the desired city
+module.exports = {
+  async details (city) {
+    const attractions = await Attractions.findAll({
+      where: { attraction_name: attraction_name },
```

Comment:
In the next sprint we will have to add logic here to recommend attractions not only based on location

Reply:
Yes you are correct